### PR TITLE
updated README for fall 2022 training

### PR DESCRIPTION
**Pull request by marianalevineu**
Updated README file for the fall 2022 training

### CODE REVIEW

**Review by joseph-cooke — COMMENTED** [review]
Line 9 has a typo - it should be Speedup

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
-# Parallel Programming on Discovery
-This repository contains material for HPC Parallel Programming on Discovery
+# HPC & Parallel Computing
+This repository contains material for HPC Parallel Computing on Discovery
 training modules.
```
Comment (manasvita) [inline README.md]: @marianalevineu , maybe we should not refer to "Discovery" here and mention something like:
This repository contains material for parallel computing on NU's high performance computing cluster

File: README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 
 ## What's in each training sub-folder?
-* Training PowerPoint slides
-* Exercise folders
+* Exercise_1 - OpenMP threads example
+* Exercise_2 - MPI with Python example (mpi4py)
+* Exercise_3 - Slurm job array example
+* Exercise_4 - Speendup, efficiency and weak scaling examples.
```
Comment (manasvita) [inline README.md]: @marianalevineu , as @joseph-cooke pointed out, it should be Speedup and not Speendup.

**Review by manasvita — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marianalevineu) [reply README.md]: thanks @manasvita  I removed the Discovery mentions as recommended.

File: README.md (hunk 2 above)
Comment (marianalevineu) [reply README.md]: thanks @joseph-cooke  @manasvita  the typo was fixed.

**Comment by marianalevineu** [comment]
@manasvita @joseph-cooke  thanks for your review, I've applied your suggestions let me know if there's anything else. thanks!

File: README.md [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+# HPC & Parallel Computing
+This repository contains material for HPC and parallel computing on NU's high performance computing cluster training modules.
 
 ## What's in each training sub-folder?
-* Training PowerPoint slides
-* Exercise folders
+* Exercise_1 - OpenMP threads example
+* Exercise_2 - MPI with Python example (mpi4py)
+* Exercise_3 - Slurm job array example
+* Exercise_4 - Speedup, efficiency and weak scaling examples.
```
Comment (dhml) [inline README.md:8]: @marianalevineu, this is a nit-pick, but the exercise 4 and 5 lines end with a period whereas the exercise 1-3 lines do not.  Consider making them all consistent.

Comment (marianalevineu) [reply README.md:8]: thanks @dhml good point, I fixed it now as well.

**Review by joseph-cooke — APPROVED** [review]

**Pull request merged**